### PR TITLE
Fix bug in GlobalRandomDroppingPoints

### PR DESCRIPTION
**Pull request by lengzq**
The drop mask shape should be [num frame, num point, 1] instead of [num frame, num point, num feature].

### CODE REVIEW

File: keras_cv/layers/preprocessing3d/global_random_dropping_points.py [hunk 1]
```diff
@@ -57,9 +57,10 @@ def get_config(self):
         return {"drop_rate": 1 - self._keep_probability}
 
     def get_random_transformation(self, point_clouds, **kwargs):
+        point_clouds_shape = point_clouds.get_shape().as_list()
         random_point_mask = (
             self._random_generator.random_uniform(
-                tf.shape(point_clouds), minval=0.0, maxval=1
+                point_clouds_shape[:-1] + [1], minval=0.0, maxval=1
```
[the remark this answers is not in the document]
Comment (ianstenbit) [reply keras_cv/layers/preprocessing3d/global_random_dropping_points.py:63]: Maybe also a comment? It's not obvious to me what exactly this means

**Review by ianstenbit — APPROVED** [review]
LGTM once a test is added

File: keras_cv/layers/preprocessing3d/global_random_dropping_points.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lengzq) [reply keras_cv/layers/preprocessing3d/global_random_dropping_points.py]: Done.

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**